src/components/grid-view/GridView.js

```diff
@@ -54,7 +53,8 @@ class GridView extends React.Component<Props> {
     }
 
     cellRenderer = ({ dataKey, parent, rowIndex }: TableCellRendererParams) => {
-        const { columnCount, count, currentCollection, slotRenderer, onItemSelect } = this.props;
+        const { columnCount, currentCollection, slotRenderer, onItemSelect } = this.props;
+        const count = currentCollection.items ? currentCollection.items.length : 0;
```

Comment:
`lodash/get` is useful here to avoid this type of defensive checking, we use it in many places as such as https://github.com/box/box-ui-elements/blob/90bf2f3c90f158068b82605aaeb66e0aa2880b64/src/elements/content-sidebar/SidebarFileProperties.js#L8